newIDE/app/public/external/piskel/piskel-main.js

```diff
@@ -347,7 +364,11 @@ ipcRenderer.on('piskel-open', (event, externalEditorInput) => {
     parentElement: pathEditorHeaderDiv,
     editorContentDocument: document,
     onSaveToGd: saveToGD,
-    onCancelChanges: closeWindow,
+    onCancelChanges: () => {
+      // Tell Piskel we saved the changes so that it does not try to prevent closing the window.
+      pskl.app.savedStatusService.onPiskelSaved();
```

Comment:
that seems like a hack 🤔  and it's quite confusing.
There's no other way to force closing the window?

Reply:
Yes it's a hack, and the alternative was to override a method in their "Unload" service. Seems even worse, while this "onPiskelSaved" is just setting a boolean so feels fairly safe... even if not pretty.